### PR TITLE
WIP: Divide sunbeam.cpp and expose deck to python

### PR DESCRIPTION
**Pull request by JensGM**

### CODE REVIEW

File: examples/output_proto.py [hunk 1]
```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env python
+
+import sys
+from os.path import isdir, join
+import sunbeam
+
+
+def main():
```
Comment (joakim-hove) [inline examples/output_proto.py]: Rename `main() -> parse(input_file)`

File: examples/output_proto.py [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+
+
+
+
+
+
+
+################################################################################
+
+def opmdatadir():
```
Comment (joakim-hove) [inline examples/output_proto.py]: No reason to go through theese hoops with `global OPMDATA_DIR` - just take the deck input from `sys.argv`:
```python
parse( sys.argv[1] )
```

File: python/sunbeam/parser.cpp [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+
+#include "parser.hpp"
+
+namespace py = boost::python;
+using namespace Opm;
+
+
+Deck parser::parseFile(const std::string& filename) {
+    ParseContext pc;
+    pc.update("PARSE_RANDOM_SLASH" , InputError::IGNORE);
```
Comment (joakim-hove) [inline python/sunbeam/parser.cpp]: This is policy - should not be in the library code.

File: python/sunbeam/properties2.py [hunk 4]
```diff
@@ -0,0 +1,8 @@
+from os.path import isfile
```
Comment (joakim-hove) [inline python/sunbeam/properties2.py]: The name `properties2` was really weird. To be able to facilitate dynamic loading of keywords you need to expose a `Parser` class also in Python - not just hook directly into the library.

**Comment by joakim-hove** [comment]
Right now we have both C++ and Python code under the `python/` directory. Would it be possible to reorganize things slightly - so that we have separate C++ and Python roots in the source tree?

**Comment by JensGM** [comment]
@joakim-hove I will clean these up, both properties2.py and output_proto.py are present only for local testing purposes and will not be included in the final commit.

**Pull request closed without merging**